llvm/test/CodeGen/RISCV/cheri/double-mem.ll

```diff
@@ -0,0 +1,75 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
```

Comment:
This is currently just a copy of float-mem other than the comment on the line below.